bpm/bonita-web-server/src/main/java/org/bonitasoft/console/common/server/filter/CacheFilter.java

```diff
@@ -84,7 +84,7 @@ private void setResponseHeader(final HttpServletResponse response) {
         final Integer duration = Integer.valueOf(paramMap.get(DURATION));
         final boolean alwaysCaching = Boolean.parseBoolean(paramMap.get(ALWAYS_CACHING));
         final boolean noCustomPageCache = Boolean.parseBoolean(System.getProperty(NO_CUSTOMPAGE_CACHE));
-        if (!noCustomPageCache || alwaysCaching) {
+        if (!noCustomPageCache && alwaysCaching) {
```

Comment:
Hi,
The variable is called "noCustomPageCache", which I understand to disable caching in development mode. We can set it in Bonita Studio to receive the latest data from the local server. So, if we want to disable caching, we set it to true, and then "!true" becomes false. Therefore, this variable does nothing because the condition with "or" depends on the settings in the web.xml file. However, if we change "or" to "and," setting noCustomPageCache to true, and then "!true" becomes false will force the disabling of caching regardless of the settings in web.xml. If it is equal to false (in a production environment, it will be equal to false), then !false is true, and caching will work according to the settings in web.xml 

Of course, it's just my suggestion, and perhaps the intended behavior of the variable "noCustomPageCache" is different.